src/GitHub.App/SampleData/PullRequestReviewFileCommentViewModelDesigner.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Reactive;
+using GitHub.ViewModels.GitHubPane;
+using ReactiveUI;
+
+namespace GitHub.SampleData
+{
+    public class PullRequestReviewFileCommentViewModelDesigner : IPullRequestReviewFileCommentViewModel
```

Comment:
Because this class isn't used directly by the designer, it's used by other sample data view models, so they set the sample data. See https://github.com/github/VisualStudio/pull/1491/files/692beca0cb8b2139246cfa8ce4b71e9e88ba037e#diff-b62ccfc33b879b821bb668fd4cdbee75R37 for example.